docs/data_management/tutorial/globus_group_tutorial.md

```diff
@@ -0,0 +1,223 @@
+
+1. Navigate to the [Globus Connect Personal page](https://www.globus.org/globus-connect-personal), scroll to look for the GCP Windows version. Click on the **`INSTALL NOW`** button highlighted red in the image below.
+
+    ![!Install GCP for Windows](./images/gcp_install_windows.png)
+
+1. Click the `Download Globus Connect Personal` link on this page, it would redirect you to download the GCP Installer.
+
+    ![!Download GCP for Windows](./images/gcp_dl_installer_windows.png)
+
+1. As shown in the image below,  click the `Download Globus Connect Personal for Windows` button preceded by the Microsoft/Windows logo to download the installer. This would download the application installer into your local computer.
+
+    ![!Download GCP Installer for Windows](./images/gcp_download_button.png)
+
+1. Navigate to where the file is downloaded on your local computer. Double click to open the downloaded file named **globusconnectpersonal-latest.exe**
+
+    ![!Downloaded GCP Installer for Windows file](./images/gcp_installer_file.png)
+
+1. Select where you would prefer to have your GCP installed (by default it installs into your C:\ drive), and click the **Install** button.
+
+    ![!Completing GCP Setup for Windows file highlighting Install button](./images/gcp_click_install.png)
+
+    Please note you must have administrator permissions, to do this. If you see the below image, please contact your local computer's administrator to complete the installation.
+
+    ![!Administrator privileges for GCP Installer](./images/gcp_admin_install.png)
+
+1. Allow the installation to complete, and click on **Finish**. This will complete the GCP installation.
+
+    ![!Completing GCP Setup highlighting Finish button](./images/gcp_click_finish.png)
+
+1. Following installation, GCP will launch in a new window, click on **Log In** to authenticate with Globus, to begin the Collection Setup process.
+
+    ![!Authenticating GCP Setup highlighting Login button](./images/gcp_globus_login.png)
+
+1. Grant the required consents to GCP setup. And include a name of your liking in the highlighted area.
+
+    ![!Granting GCP Setup required permissions](./images/gcp_consent.png)
+
+1. Enter the details for your GCP Collection, and click save to continue. In the image below, **1** should contain your preferred identity, if using a UAB Computer, please insert your `BlazerID@uab.edu` email address here. **2** Should contain the name for your Collection, already named in the previous step. **3** Should contain optional information to describe the collection, you can include the purpose for which the collection has been created. **4** If your collection would contain Protected Health Information (PHI) or sensitive information, please tick this box. You would need to be part of UAB's subscription to complete this, please send an email to support, so we can complete that process on your behalf.
```

Comment:
I think the **bold** choice here for the numbers is good. But let's go one step further and make the elements a sublist.

1. Enter the details
    1. Item 1
    1. Item 2
    1. etc...

Reply:
If they aren't numeric, like here in GitHub, then we can use a bulleted list followed by **(1)**.

Try to format them like in the image, the parentheses `()` give it a round appearance. So putting it all together...

- **(1)**: description
- **(2)**: description

Reply:
Made these changes